packages/@sanity/cli/test/dev.test.ts

```diff
@@ -105,11 +105,11 @@ describeCliTest('CLI: `sanity dev`', () => {
         basePath: '/app-base-path',
         args: ['--port', `${port}`],
         cwd: path.join(fixturesPath, 'app'),
-        expectedTitle: 'Sanity CORE App',
+        expectedTitle: 'Sanity SDK App',
```

Comment:
Did you mean to just change the assertion? Think the command that outputs this also needs updating